templates/taxi/car_confirm_delete.html

```diff
@@ -6,5 +6,6 @@ <h1>Delete car?</h1>
     {% csrf_token %}
 
     <input type="submit" value="Yes" class="btn btn-danger">
+    <a href="{% url 'taxi:car-detail' pk=car.id %}" class="btn btn-secondary">Cancel</a>
```

Comment:
Ensure that the context variable `car` is passed to this template from the view. This is necessary for the `{% url 'taxi:car-detail' pk=car.id %}` tag to work correctly.